wordpress-mobile/WordPress-iOS
# Allow inserting links to documents from Media Library

This PR allows users to insert non-video/image files into a post using the full screen media library picker. For now, we are just inserting a link using the media item title as the link name (defaulting to URL if the title is empty).

![insert_doc_links](https://user-images.githubusercontent.com/154014/30968079-b79544fe-a423-11e7-9fba-8ac3ae9dac38.gif)

Note that we are not using shortcodes for audio...that will be addressed in #7862 
 
Fixes #7689

To test:

0. Upload some non-image/video files (.PDF, .DOC, .MP3, etc) to your .org/.com site
1. Insert the media into a post and verify the links are correct.

Needs review: @SergioEstevao would you mind?
